docs/plugins.md

```diff
@@ -176,21 +176,28 @@ function print(
   path: AstPath,
   options: object,
   // Recursively print a child node
-  print: (path: AstPath) => Doc
+  printGenerically: (
```

Comment:
If we want to change this in the docs, we need a better name than `printGenerically`.

Reply:
How about `printProperty`? Any better idea?

Reply:
It's passed around so much that I believe `print` is the best name, even though this name coincides with `parser.print` (I guess that's the reason why you want to rename it).